daras_ai_v2/base.py

```diff
@@ -490,8 +486,17 @@ def _render_unpublished_changes_indicator(self):
             gui.html("Unpublished changes")
 
     def _render_saved_timestamp(self, pr: PublishedRun):
```

Comment:
use the current_sr / current_pr here instead of passing down the pr and using statekeys.updated_at